ts/components/bottomSheet/BlurredBackgroundComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import TouchableDefaultOpacity from "../TouchableDefaultOpacity";
+
+export const BlurredBackgroundComponent = (onPress: ()=> void) => (
```

Comment:
```suggestion
export const BlurredBackgroundComponent = (onPress: ()=> void, backGroundAlhpa: number = 0.5) => (
```
Could it make this component more flexible?

Reply:
@Undermaken do you mean then changing the `backgroundColor` as follows:

```
backgroundColor: `rgba(0,0,0, ${backGroundAlhpa})`
```
it surely adds a new editale configuration, but maybe we should handle the color opacity globally in order to avoid mixed solutions (in case we're going to reuse this component elsewhere from BottomSheet

what do you think?

Reply:
@CrisTofani 
Yes, it's a good observation of [YAGNI](https://it.wikipedia.org/wiki/You_aren%27t_gonna_need_it)
We'll add if it will be a need